src/plugins/wb-data-scrub/wb-data-scrub.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @title WET-BOEW wb-data-scrub
+ * @overview This plugin delete Personal Identifiable Information (PII) from the flagged form fields before form submit
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @polmih, @duboisp, @GormFrank
+ **/
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-data-scrub",
+	selector = "[data-rule-wbscrub]",
+	initEvent = "wb-init" + componentName,
+	wrapper,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		if ( elm && $( elm.form ).data( "scrub-inited" ) !== "" ) {
+
+			var	form = elm.form;
+
+			$( elm.form ).data( "scrub-inited", "" );
+
+			form.addEventListener( "submit", function( event ) {
+				event.preventDefault();
+				var isPii = false,
+					moDalId = componentName + "-modal",
+					i18nDict = {
+						en: {
+							"msgboxHeader": "Remove Personal information",
+							"msgBodyIntro": "Some information added in the following fields is identified as personal information and it will be removed and submitted as follows:",
+							// eslint-disable-next-line max-len
+							"msgBodyViewMore": "<p>Please note that any information that corresponds to the cases from the list below, is considered as personal information and is removed from the form prior its submission:</p> <ul><li>Any 8 or more digits separated or not by the following characters: ' ', '/', '-', '.'</li><li>Passport serial number</li><li>Email</li><li>Postal Code</li><li>Certain parametres considered non secured in a url</li></ul>",
+							"yesBtn": "I agree, remove personal information and submit",
+							"cancelBtn": "Go back to the form",
+							"viewMoreLink": "Find out more"
+						},
+						fr: {
+							"msgboxHeader": "Supprimer les informations personnelles",
+							"msgBodyIntro": "Certaines informations ajoutées dans les champs suivants sont identifiées comme des informations personnelles et seront supprimées et soumises comme suit:",
+							// eslint-disable-next-line max-len
+							"msgBodyViewMore": "<p>Veuillez noter que toute information correspondant aux cas de la liste ci-dessous est considérée comme une information personnelle et est supprimé du formulaire avant sa soumission:</p> <ul><li>8 chiffres ou plus séparés ou non par les caractères suivants : ' ', '/', '-', '.'</li><li>Numéro de série du passeport</li><li>E-mail</li><li>Code Postal</li><li>Des paramètres considérés comme non sécurisés dans une url</li></ul>",
+							"yesBtn": "J'accepte, supprime les informations personnelles et soumettre",
+							"cancelBtn": "Retourner au formulaire",
+							"viewMoreLink": "En savoir plus"
+						}
```

Comment:
Those i18n would need to be added into our i18n csv file

Reply:
Done. Thanks